### PR TITLE
Update Clinvar SV doc

### PR DESCRIPTION
**Pull request by N-Ouyang**

### CODE REVIEW

File: docs/data-sources/clinvar.mdx [hunk 1]
```diff
@@ -79,9 +79,9 @@ The Acc and Version fields are merged to form the ID (RCV000000001.2)
 ```
 We only use the field with Type="Preferred". Multiple phenotypes may be reported
 
-**Location and Variant Id**
+**Location, Variant Type and Variant Id**
```
Comment (Fahd-Siddiqui) [inline docs/data-sources/clinvar.mdx:82]: We should mention the variantTypes that are skipped:
1. Microsatellite
1. protein only
1. fusion
1. Complex
1. Variation
1. Translocation

File: docs/data-sources/clinvar.mdx [hunk 2]
```diff
@@ -106,6 +106,14 @@ We only use the field with Type="Preferred". Multiple phenotypes may be reported
...
 - Only variants having a dbSNP id are extracted.
 - Note that a ClinVar accession may have multiple variants associated with it (possible in different locations)
+- VariantType is extracted from the Measure attributes.
+- Note that we currently don't support the following variant types:
+  - Microsatellite
+  - protein only
+  - fusion
+  - Complex
+  - Variation
+Translocation
```
Comment (Fahd-Siddiqui) [inline docs/data-sources/clinvar.mdx]:
```suggestion
- VariantType is extracted from the Measure attributes.
- Note that we currently don't support the following variant types:
  - Microsatellite
  - protein only
  - fusion
  - Complex
  - Variation
  - Translocation
```

Comment (Fahd-Siddiqui) [reply docs/data-sources/clinvar.mdx]: @rajatshuvro Should this be here or under known issues?

File: docs/data-sources/clinvar.mdx [hunk 3]
```diff
@@ -106,6 +106,14 @@ We only use the field with Type="Preferred". Multiple phenotypes may be reported
 - If a required allele is not available, we extract it from the reference sequence.
 - Only variants having a dbSNP id are extracted.
 - Note that a ClinVar accession may have multiple variants associated with it (possible in different locations)
+- VariantType is extracted from the Measure attributes.
+- Note that we currently don't support the following variant types:
```
Comment (rajatshuvro) [inline docs/data-sources/clinvar.mdx]: I would add this under a warning like:

:::caution Varying Delimiters
The delimiters in each field may vary. Currently, the delimiters for `Description` are `,` and `/`. The delimiters for `Explanation` are `;` and `/`.
:::

File: docs/data-sources/clinvar.mdx (hunk 2 above)
Comment (N-Ouyang) [reply docs/data-sources/clinvar.mdx]: nice catch

**Review by rajatshuvro — APPROVED** [review]

**Review by Fahd-Siddiqui — APPROVED** [review]

**Pull request merged**